app/lib/actions/pulp/orchestration/repository/remove_units.rb

```diff
@@ -0,0 +1,15 @@
+module Actions
+  module Pulp
+    module Orchestration
+      module Repository
+        class RemoveUnits < Pulp::Abstract
+          def plan(_repository, _smart_proxy, options)
+            sequence do
+              plan_action(Actions::Repository::RemoveUnits, options)
+            end
```

Comment:
You probably meant `Actions::Pulp::Repository::RemoveUnits` here..

Reply:
You mean, without the `sequence` term?

Reply:
The namespace seems to be off here.. The remove units action would be under pulp module?
Also, the sequence do block can be ommited for a singular action.